src/ShopBundle/objects/db/TShopPaymentHandler/TShopPaymentHandlerPayPal_PayViaLink.class.php

```diff
@@ -240,7 +271,12 @@ public function ProcessIPNRequest($oOrder, $aURLParameter)
         }
 
         if (false == $bPaymentOk) {
-            $logger->error('PayPal IPN: payment invalid '.print_r($aURLParameter, true));
+            $logger->error(
+                'PayPal IPN: payment invalid.',
+                [
+                    'url_parameters' => $aURLParameter
```

Comment:
please add order id and ordernumber to the context

Reply:
ok